dotcom-rendering/src/lib/assets.ts

```diff
@@ -25,7 +25,7 @@ export const decideAssetOrigin = (
 		case 'CODE':
 			return 'https://assets-code.guim.co.uk/';
 		default: {
-			if (isDev) {
+			if (isDev && process.env.HOSTNAME === undefined) {
```

Comment:
Maybe leverage `@guardian/libs` here?

```suggestion
			if (isDev && isUndefined(process.env.HOSTNAME)) {
```

Reply:
Good shout